README.md

```diff
@@ -307,6 +307,12 @@ flank:
 
   ## Local folder to store the test result. Folder is DELETED before each run to ensure only artifacts from the new run are saved.
   # local-result-dir: flank
+    
+  ## Include additional app/test apk pairs in the run. If app is omitted, then the top level app is used for that pair.
+  # additional-app-test-apks:
+  #  - app: ../test_app/apks/app-debug.apk
+  #    test: ../test_app/apks/app-debug-androidTest.apk
```

Comment:
Have you thought of making `test` a list, so you don't have a bunch of usages without `app`?

Something like

```yml

- app: ../app.apk
  test:
    - testapk1.apk
    - testapk2.apk

- app: app2.apk
  test:
     - testfinal.apk
```

Reply:
I like that idea.

Reply:
I prototyped adding list support, in particular this complicated the code when parsing from CLI. 

- https://github.com/remkop/picocli/issues/677

I think for the first version of his feature, the more verbose format is easier to maintain. The test run loop also becomes more complex as we have yet another list to iterate through.

```yaml
- app: ../app.apk
  test: test1.apk
- app: ../app.apk
  test: test2.apk
```

I'm open to revisiting this based on feedback in the future.